src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1889,9 +1932,28 @@ namespace HeatBalFiniteDiffManager {
 
                 // Source/Sink Flux Capability ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-                Real64 const QSSFlux((surface.Area > 0.0) && (construct.SourceSinkPresent && Lay == construct.SourceAfterLayer)
-                                         ? (state.dataHeatBalFanSys->QRadSysSource(Surf) + state.dataHeatBalFanSys->QPVSysSource(Surf)) / surface.Area
-                                         : 0.0); // Source/Sink flux value at a layer interface // Includes QPV Source
+                Real64 QSSFlux = 0.0;
+                if ((surface.Area > 0.0) && (construct.SourceSinkPresent && Lay == construct.SourceAfterLayer)) {
+                    // Source/Sink flux value at a layer interface // Includes QPV Source
+                    QSSFlux = (state.dataHeatBalFanSys->QRadSysSource(Surf) + state.dataHeatBalFanSys->QPVSysSource(Surf)) / surface.Area;
+                }
+
+                // Add EMS actuated value
+                if (heatFluxActuator.isActuated) {
+                    Real64 actuatedVal = heatFluxActuator.actuatedValue;
+                    if (actuatedVal >= 0) {
+                        QSSFlux += heatFluxActuator.actuatedValue;
+                    } else {
+                        ShowSevereError(state, fmt::format("Surface: {}, Material: {}", surface.Name, mat.Name));
+                        ShowContinueError(state, "EMS Actuator does not support negative values");
+                        ShowFatalError(state, "Program terminates due to preceding conditions.");
+                    }
+
+                    // Update report variables
+                    auto &surfFD = state.dataHeatBalFiniteDiffMgr->SurfaceFD(Surf);
+                    surfFD.heatSourceFluxLayerReport(Lay) = QSSFlux * surface.Area;
+                    surfFD.heatSourceFluxEnergyLayerReport(Lay) = QSSFlux * surface.Area * state.dataGlobal->TimeStepZoneSec;
+                }
```

Comment:
Acuating here, which is an addition to whatever is also occurring due to the `InternalHeatSource` object.